### PR TITLE
Kanban perspective #LMR-1664

### PR DESCRIPTION
**Pull request by kubedo8**

### CODE REVIEW

File: src/app/shared/collapsible-sidebar/collapsible-sidebar.component.scss [hunk 1]
```diff
@@ -9,6 +9,10 @@
   overflow: auto;
 }
 
+.toggle {
+  opacity: 0.6;
```
Comment (marvec) [inline src/app/shared/collapsible-sidebar/collapsible-sidebar.component.scss]: I suggest to have opacity 1.0 when hover. WDYT?

File: src/app/view/perspectives/chart/chart-perspective.component.ts [hunk 2]
```diff
@@ -146,7 +146,7 @@ export class ChartPerspectiveComponent implements OnInit, OnDestroy {
   private subscribeData() {
     this.documents$ = this.store$.pipe(
       select(selectDocumentsByQuery),
-      distinctUntilChanged((x, y) => JSON.stringify(x) === JSON.stringify(y))
+      distinctUntilChanged((x, y) => deepObjectsEquals(x, y))
     );
```
Comment (marvec) [inline src/app/view/perspectives/chart/chart-perspective.component.ts]: You could now use isEqual() from lodash (which also has isDeepEquals).

Comment (kubedo8) [reply src/app/view/perspectives/chart/chart-perspective.component.ts]: `deepObjectsEquals` uses `isEqual()` from lodash. You did it recently ;)

File: src/app/shared/collapsible-sidebar/collapsible-sidebar.component.scss (hunk 1 above)
Comment (kubedo8) [reply src/app/shared/collapsible-sidebar/collapsible-sidebar.component.scss]: Ok

File: src/app/view/perspectives/chart/chart-perspective.component.ts (hunk 2 above)
Comment (marvec) [reply src/app/view/perspectives/chart/chart-perspective.component.ts]: I see, it is the other way around!

**Pull request merged**